core/vm/evm.go

```diff
@@ -234,9 +238,7 @@ func (self *EVM) Main(trx *Transaction) (ret ExecutionResult) {
 	}
```

Comment:
isnt this if else the place where the tx is samhow saved in db or tx is saved before ?

Reply:
It is just contract code execution part. There is no code related to saving inside of call\create